index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
         <img src="#" alt="FontIcon" class="font-icon" />
         <img src="#" alt="OtherImg" class="fourth-illustration" />
       </figure>
-      <section class="product-container">
-        <ul class="product-list">
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
-          </li>
-          <li class="product-item">
-            <img src="#" alt="ProductImg" class="product-img" />
+    </main>
+    <section class="product-container">
+      <ul class="product-list">
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+        <li class="product-item">
+          <img src="#" alt="ProductImg" class="product-img" />
+        </li>
+      </ul>
+    </section>
+    <section class="spotlight-container">
+      <div class="spotlight-header">
+        <h2 class="spotlight-heading">Spotlight Digital Assets Selections</h2>
+        <a href="#" class="explore-featured">Explore Featured Works &gt;</a>
+      </div>
+      <div class="spotlight-product-container">
+        <div class="spotlight-product">
```

Comment:
Please use tag `article` for reused blocks 

The `<article>` tag is used to represent a self-contained piece of content that could be independently distributed or reused.
Each `<article>` should make sense on its own, even when taken out of context.

**Usage Examples:**
- Blog posts
- News articles
- Forum posts
- User comments
- Product cards

**Characteristics:**
- The content within an `<article>` is independent and complete.
- Multiple `<article>` elements can be used on a single page to represent different pieces of self-contained content.